containers/base/javascript/browser/css/dashboard.css

```diff
@@ -56,3 +179,631 @@ progress[value]::-moz-progress-bar {
 	font-family: Lato;
 	font-size: 13px;
 }
+
+.sending-interface{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.sending-interface ol {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.sending-interface button {
+    cursor: pointer;
+    margin-bottom: var(--spacing-medium);
+}
+
+.sending-interface .dropzone-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    min-height: 10em;
+    width: 40em;
+    border-radius: var(--border-radius-small);
+    border: 2px black dashed;
+    margin-top: var(--spacing-medium);
+    margin-bottom: var(--spacing-medium);
+}
+
+.sending-interface .cardcontainer{
+    display: flex;
+}
+
+.sending-interface .cardcontainer .card{
+    display: flex;
+    margin: var(--spacing-small);
+}
+
+.sending-interface .explanation{
+    width: 40em;
+}
+
+.sending-interface .dropzone-container .dropzone-info{
+    display: flex;
+    margin: var(--spacing-medium);
+}
+
+.sending-interface .dropzone-container .dropzone-info .link_button{
+    border: none;
+    color: var(--premium-blue);
+    font-weight: 700;
+    font-size: 14px;
+    margin: 0px;
+    padding: 0px;
+}
+
+.sending-interface .dropzone-container .dropzone-info .text{
+    margin: var(--spacing-small);
+}
+
+.sending-interface .dropzone-container .dropzone-info .image{
+    width: 4em;
+    height: 4em;
+}
+
+.sending-interface .explanation-container{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.sending-interface .explanation-container .step-container{
+    display: flex;
+    flex-direction: column;
+    width: 100px;
+}
+
+.sending-interface .explanation-container .step-container p{
+    text-align: center;
+    line-height: 1em;
+}
+
+.sending-interface .explanation-container .step-container .step{
+    height: 100px;
+    width: 100px;
+}
+
+.sending-interface .explanation-container .pointer{
+    height: 30px;
+    width: 30px;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.general-controls{
+    display: inline-block;
+    vertical-align: baseline;
+}
+
+/* todo: hover color only when file is draged over;
+/*.drop-fonts:hover {
+    background: var(--premium-light-blue);
+}*/
+
+.filename {
+    white-space: nowrap;
+    display: block;
+    padding-left: 1em;
+}
+
+/* Reporting Interface */
+
+.report-container_isFinished > strong,
+.report-container_created > strong,
+.report-container_started > strong,
+.report-container_finished > strong {
+    display: inline-block;
+    margin-right: .5em;
```

Comment:
If you want whitespace on an inline element, you can also use padding and not set `display`